Polyhedron/demo/Polyhedron/Scene_polyhedron_item.h

```diff
@@ -62,7 +62,6 @@ class SCENE_POLYHEDRON_ITEM_EXPORT Scene_polyhedron_item
     bool load_obj(std::istream& in);
     bool save(std::ostream& out) const;
     bool save_obj(std::ostream& out) const;
-
```

Comment:
If you re-add that empty line in the file, then the `.h` will not be modified, and that will avoid to recompile all plugins that depend on the polyhedron item.